apps/daengle/src/pages/login/index.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from './index.module.scss';
+
+export default function Login(): JSX.Element {
```

Comment:
여기에 타입은 굳이 지정 안 해도 될 거 같아용
```suggestion
export default function Login() {
```

Reply:
error가 떠서 추가한 부분인데 무시해도 될까요 ?
eslint에 설정 추가하면 될까요 ?

Reply:
어떤 에러가 떴나요?!